sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/RpAttachType.java

```diff
@@ -9,7 +9,7 @@ public enum RpAttachType {
     STANDALONE_AUTO("StandaloneAuto"), // RP attach is enabled via a custom JAVA_OPTS or on premise resources
     INTEGRATED_AUTO("IntegratedAuto"); // RP attach is on by default
```

Comment:
Maybe INTEGRATED_AUTO could be more explicit? Does it mean the Application Insights Java agent is used with` java -javaagent`?

Reply:
same here.. it's straight from the spec.